sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/contacts/EditContactSteps.java

```diff
@@ -98,6 +102,145 @@ public EditContactSteps(WebDriverHelpers webDriverHelpers) {
+              .isEqualToIgnoringCase(editedContact.getProhibitionToWork());
+          softly
+              .assertThat(aContact.getHomeBasedQuarantinePossible())
+              .isEqualToIgnoringCase(editedContact.getHomeBasedQuarantinePossible());
+          softly
+              .assertThat(aContact.getQuarantine())
+              .isEqualToIgnoringCase(editedContact.getQuarantine());
+          softly
+              .assertThat(aContact.getHighPriority())
+              .isEqualToIgnoringCase(editedContact.getHighPriority());
+          softly
+              .assertThat(aContact.getPreexistingConditionDiabetes())
+              .isEqualToIgnoringCase(editedContact.getPreexistingConditionDiabetes());
+          softly
+              .assertThat(aContact.getPreexistingConditionHiv())
+              .isEqualToIgnoringCase(editedContact.getPreexistingConditionHiv());
+          softly
+              .assertThat(aContact.getPreexistingConditionLiver())
+              .isEqualToIgnoringCase(editedContact.getPreexistingConditionLiver());
+          softly
+              .assertThat(aContact.getPreexistingConditionMalignancy())
+              .isEqualToIgnoringCase(editedContact.getPreexistingConditionMalignancy());
+          softly
+              .assertThat(aContact.getPreexistingConditionChronicPulmonary())
+              .isEqualToIgnoringCase(editedContact.getPreexistingConditionChronicPulmonary());
+          softly
+              .assertThat(aContact.getPreexistingConditionRenal())
+              .isEqualToIgnoringCase(editedContact.getPreexistingConditionRenal());
+          softly
+              .assertThat(aContact.getPreexistingConditionNeurologic())
+              .isEqualToIgnoringCase(editedContact.getPreexistingConditionNeurologic());
+          softly
+              .assertThat(aContact.getPreexistingConditionCardiovascular())
+              .isEqualToIgnoringCase(editedContact.getPreexistingConditionCardiovascular());
+          softly
+              .assertThat(aContact.getAdditionalRelevantPreexistingConditions())
+              .isEqualToIgnoringCase(editedContact.getAdditionalRelevantPreexistingConditions());
+          softly
+              .assertThat(aContact.getVaccinationStatusForThisDisease())
+              .isEqualToIgnoringCase(editedContact.getVaccinationStatusForThisDisease());
+          softly
+              .assertThat(aContact.getImmunosuppressiveTherapy())
+              .isEqualToIgnoringCase(editedContact.getImmunosuppressiveTherapy());
+          softly
+              .assertThat(aContact.getActiveInCare())
+              .isEqualToIgnoringCase(editedContact.getActiveInCare());
+          softly
+              .assertThat(aContact.getOverwriteFollowUp())
+              .isEqualToIgnoringCase(editedContact.getOverwriteFollowUp());
+          // softly
+          //    .assertThat(aContact.getDateOfFollowUpUntil())
+          //    .isEqualTo(editedContact.getFollowUpStatusComment());
```

Comment:
Please remove the code or let the ticket where a bug is created

Reply:
https://github.com/hzi-braunschweig/SORMAS-Project/issues/5634

Reply:
rezolved imho